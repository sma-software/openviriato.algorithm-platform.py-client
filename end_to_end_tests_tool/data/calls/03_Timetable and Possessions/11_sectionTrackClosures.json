{
    "restResource": "/possessions/section-track-closures",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "FromTime",
            "value": "2003-05-01T07%3A30%3A00",
            "optional": false
        },
        {
            "key": "ToTime",
            "value": "2003-05-01T08%3A05%3A45",
            "optional": false
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetSectionTrackClosures(timeWindow)",
    "CSharpMethodParameterTypes": [ "SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.ITimeWindow" ],
    "subcategory": "Get Section Track Closures"
}