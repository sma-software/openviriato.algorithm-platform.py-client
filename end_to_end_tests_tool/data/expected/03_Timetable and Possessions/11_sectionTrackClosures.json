[
  {
    "debugString": "sectiontrackclosure:s_70011 1 n_85ZMUS 85ZLST A",
    "sectionTrackId": 1080,
    "fromNodeId": 621,
    "toNodeId": 620,
    "closureTimeWindowFromNode": {
      "fromTime": "2003-05-01T08:00:00",
      "toTime": "2003-05-01T09:00:00"
    },
    "closureTimeWindowToNode": {
      "fromTime": "2003-05-01T08:30:00",
      "toTime": "2003-05-01T09:30:00"
    }
  },
  {
    "debugString": "sectiontrackclosure:s_70011 0 n_85ZMUS 85ZLST A",
    "sectionTrackId": 1079,
    "fromNodeId": 621,
    "toNodeId": 620,
    "closureTimeWindowFromNode": {
      "fromTime": "2003-05-01T08:00:00",
      "toTime": "2003-05-01T09:00:00"
    },
    "closureTimeWindowToNode": {
      "fromTime": "2003-05-01T08:30:00",
      "toTime": "2003-05-01T09:30:00"
    }
  }
]